### PR TITLE
Internal: Undo action breaking Accordion Nested Element on the editor [ED-14309]

### PR DESCRIPTION
**Pull request by baghdasarovelementor**
## Description
Internal: Undo action breaking Accordion Nested Element on the editor 

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities

### CODE REVIEW

**Review by louiswol94 — APPROVED** [review]
So this means that on Undo, we should just re-render the whole widget again?

**Review by davseve — APPROVED** [review]

File: assets/dev/js/editor/document/repeater/commands/insert.js [hunk 1]
```diff
@@ -14,12 +14,14 @@ export class Insert extends $e.modules.editor.document.CommandHistoryBase {
 				model: data.model,
 				name: data.name,
 				options: { at: data.index },
+				isRestored: true,
 			} );
 		} else {
 			$e.run( 'document/repeater/remove', {
```
Comment (davseve) [inline assets/dev/js/editor/document/repeater/commands/insert.js:20]: This is the undo?

Comment (baghdasarovelementor) [reply assets/dev/js/editor/document/repeater/commands/insert.js:20]: Yes

**Comment by baghdasarovelementor** [comment]
@louiswol94 Yes

**Comment by baghdasarovelementor** [comment]
> So this means that on Undo, we should just re-render the whole widget again?

To fix this issue without re-render of the whole widget we need much more time and resources, so I implemented it like this for now, we can make it work better later

**Pull request merged**